src/library-authoring/import-course/stepper/ReviewImportDetails.tsx

```diff
@@ -114,15 +115,26 @@ const Banner = ({ courseId, isBlockDataPending, unsupportedBlockPercentage }: Ba
   );
 };
 
-export const ReviewImportDetails = ({ courseId, markAnalysisComplete }: Props) => {
+export const ReviewImportDetails = ({
+  courseId,
+  markAnalysisComplete,
+  setImportIsBlocked,
+}: Props) => {
   const { data: blockTypes, isPending: isBlockDataPending } = useGetBlockTypes([
     `context_key = "${courseId}"`,
   ]);
+  const {
+    data: libraryBlockLimits,
+    isPending: isPendinglibraryBlockLimits,
+  } = useLibraryBlockLimits();
+  const {
+    libraryData,
+  } = useLibraryContext();
```

Comment:
The `num_blocks` field from `libraryData` only returns the number of component blocks and does not include containers in it.

You can replace this call with something like 

```tsx
const { libraryId } = useLibraryContext();

const { data: libBlockTypes, isPending: isLibBlockDataPending } = useGetBlockTypes([
  `context_key = "${libraryId}"`,
]);

// And get count by adding all block types in `libBlockTypes`
```